src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -29,12 +30,17 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
     new_is_better: newIsBetter,
   } = result;
 
+  const NumberFormat = (value: number) => {
+    return new Intl.NumberFormat('en-US', {
+      maximumFractionDigits: 2,
+    }).format(value);
```

Comment:
Please create Intl.NumberFormat outside of the component, so that the instance is reused for each render. It's very very important for performance.

Reply:
Okay noted